ui/prometheus-plugin/src/utils/utils.test.ts

```diff
@@ -160,10 +160,18 @@ describe('getFormattedPrometheusSeriesName', () => {
   it('should resolve empty metric to instead show query', () => {
     const query = 'node_load15{instance=~"(demo.do.prometheus.io:9100)"';
     const metric = {};
-    const output = { name: query };
+    const output = { name: query, formattedName: query };
     expect(getFormattedPrometheusSeriesName(query, metric)).toEqual(output);
   });
 
+  it('should resolve empty metric with formatted series name', () => {
+    const query = 'up';
+    const metric = {};
+    const series_name_format = 'Custom series name';
```

Comment:
Should this kick in regardless of whether there are labels? 

Maybe that's how the logic works, I haven't checked logic yet, I'm just reading the test and thinking: If someone types `my custom series name` into the input, that should show up regardless of whether the metric has labels or not, right?

Reply:
Yeah me double check the tests, but that is what this is doing right? If `series_name_format` is 'Custom series name' then the output has 'Custom series name' as the formattedName.

formattedName is then what's displayed, but I kept around the unformatted name just in case we want to show it on hover or some other use case in the future

Reply:
Ok sweet that sounds good -- I think I just wasn't sure because the test is mentioning an empty metric, and I wasn't sure if that was a requirement for this behavior

Reply:
Oh yeahh the wording is confusing, that is expected behavior for anytime series_name_format is defined. I changed the test cases around to make it more clear